### PR TITLE
Support erc20 balance

### PR DESCRIPTION
**Pull request by qwer951123**

### CODE REVIEW

File: packages/sdk/src/wallet/balance-adapter/acala.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+import { AccountInfo } from '@polkadot/types/interfaces/system';
+import { Provider as BodhiProvider } from '@acala-network/bodhi';
+
+import { OrmlAccountData } from '@open-web3/orml-types/interfaces';
+import { Contract, utils } from 'ethers';
+import { map } from 'rxjs/operators';
+import { createStorages } from '../storages';
+import { BalanceData } from '../type';
+import { BalanceAdapter } from './types';
+import ERC20_JSON from '@acala-network/contracts/build/contracts/Token.json';
```
Comment (zjb0807) [inline packages/sdk/src/wallet/balance-adapter/acala.ts:12]:
```
export const ERC20_ABI = [
  // Read-Only Functions
  'function balanceOf(address owner) view returns (uint256)',
  'function decimals() view returns (uint8)',
  'function symbol() view returns (string)',


  // Authenticated Functions
  'function transfer(address to, uint amount) returns (bool)',


  // Events
  'event Transfer(address indexed from, address indexed to, uint amount)'
];
```

Comment (zjb0807) [reply packages/sdk/src/wallet/balance-adapter/acala.ts:12]: https://github.com/AcalaNetwork/bodhi.js/blob/8f2755541a5a50edf0a83219c00eea4462fedf0a/eth-providers/src/consts.ts#L29-L40

Comment (qwer951123) [reply packages/sdk/src/wallet/balance-adapter/acala.ts:12]: ok

File: packages/sdk/package.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
     "@acala-network/api": "4.0.4-5",
     "@acala-network/bodhi": "^2.4.0",
     "@acala-network/contracts": "^3.2.0",
+    "@acala-network/eth-providers": "^2.4.0",
```
Comment (zjb0807) [inline packages/sdk/package.json:21]:
```suggestion
```

**Pull request merged**